ci-operator/jobs/openshift/machine-config-operator/openshift-machine-config-operator-master-presubmits.yaml

```diff
@@ -463,6 +463,71 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-aws-single-node,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
ideally i'd also like this to be 

always_run: false

since the other 2 single node tests are intermittently green and cause a lot of noise in our repo.

Reply:
fixed